.github/actions/run-workspace-tests/action.yml

```diff
@@ -17,6 +17,16 @@ runs:
   using: "composite"
 
   steps:
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v3
+      with:
+        role-to-assume: arn:aws:iam::${{ env.AWS_ACCOUNT }}:role/${{ env.AWS_ROLE_NAME }}
+        aws-region: ${{ env.AWS_REGION }}
```

Comment:
I don't like that much that this is taking env variables define elsewhere, I think it would be better to make them inputs and pass them from the calling workflow.